### PR TITLE
Gold report

### PR DESCRIPTION
**Pull request by ernfrid**
This brings in additional changes from the ad-hoc space needed to get useful gold reports. @indraniel - Can you review and ensure I have brought over all the necessary changes?

### CODE REVIEW

**Comment by indraniel** [comment]
Yup, these changes look fine.  They correspond to my config edits that I made for `genome-3590` in [BIO-1491](https://jira.gsc.wustl.edu/browse/BIO-1491).

```
$ diff -u gold_germline_report_indels.yaml.dlarson gold_germline_report_indels.yaml.idas
--- gold_germline_report_indels.yaml.dlarson	2015-09-04 13:01:46.000000000 -0500
+++ gold_germline_report_indels.yaml.idas	2015-09-04 12:58:19.000000000 -0500
@@ -9,7 +9,6 @@
         insertion_centric: 1
     vep:
         ensembl_version: 75
-        reference_version: GRCh37
         custom_annotation_tags:
             - ON_TARGET
             - SEG_DUP
@@ -22,7 +21,6 @@
         plugins_version: 1
         joinx_version: 1.9
         short_name: 0
-        allow_same_file: 0
     dbsnp:
         joinx_version: 1.10
         info_string: 'CAF:dbSNPBuildID=dbSNPBuildID,per-alt:MUT'


$ diff -u gold_germline_report_snvs.yaml.dlarson gold_germline_report_snvs.yaml.idas
--- gold_germline_report_snvs.yaml.dlarson	2015-09-04 13:02:50.000000000 -0500
+++ gold_germline_report_snvs.yaml.idas	2015-09-04 12:58:37.000000000 -0500
@@ -9,7 +9,6 @@
         insertion_centric: 0
     vep:
         ensembl_version: 75
-        reference_version: GRCh37
         custom_annotation_tags:
             - ON_TARGET
             - SEG_DUP


$ diff -u gold_somatic_report_snvs.yaml.dlarson gold_somatic_report_snvs.yaml.idas
--- gold_somatic_report_snvs.yaml.dlarson	2015-09-04 13:03:36.000000000 -0500
+++ gold_somatic_report_snvs.yaml.idas	2015-09-04 13:00:24.000000000 -0500
@@ -9,7 +9,6 @@
         insertion_centric: 0
     vep:
         ensembl_version: 75
-        reference_version: GRCh37
         custom_annotation_tags:
             - ON_TARGET
             - SEG_DUP
@@ -22,7 +21,6 @@
         plugins_version: 1
         joinx_version: 1.9
         short_name: 0
-        allow_same_file: 0
     dbsnp:
         joinx_version: 1.10
         info_string: 'CAF:dbSNPBuildID=dbSNPBuildID,per-alt:MUT'


$ diff -u gold_somatic_report_indels.yaml.dlarson gold_somatic_report_indels.yaml.idas
--- gold_somatic_report_indels.yaml.dlarson	2015-09-04 13:03:15.000000000 -0500
+++ gold_somatic_report_indels.yaml.idas	2015-09-04 13:00:10.000000000 -0500
@@ -9,7 +9,6 @@
         insertion_centric: 1
     vep:
         ensembl_version: 75
-        reference_version: GRCh37
         custom_annotation_tags:
             - ON_TARGET
             - SEG_DUP
@@ -22,7 +21,6 @@
         plugins_version: 1
         joinx_version: 1.9
         short_name: 0
-        allow_same_file: 0
     dbsnp:
         joinx_version: 1.10
         info_string: 'CAF:dbSNPBuildID=dbSNPBuildID,per-alt:MUT'
```

The additional `reference_version` and `allow_same_file` attributes you notice in the diffs above were added to the config files sometime after `genome-3590`.  

* The `reference_version` addition came from @apregier  in commit [29f06c09](https://github.com/genome/genome/commit/29f06c09)
* The `allow_same_file` additional came from @susannasiebert in commit [c1c61928](https://github.com/genome/genome/commit/c1c61928)

**Comment by indraniel** [comment]
+1

File: lib/perl/Genome/VariantReporting/plan_files/gold_germline_report_snvs.yaml [hunk 1]
```diff
@@ -22,15 +22,10 @@ experts:
         plugins_version: 1
         joinx_version: 1.9
         short_name: 0
-        allow_same_file: 0
```
Comment (apregier) [inline lib/perl/Genome/VariantReporting/plan_files/gold_germline_report_snvs.yaml]: Did you also intend to remove this parameter from gold_germline_report_indels.yaml ?

Comment (apregier) [inline lib/perl/Genome/VariantReporting/plan_files/gold_germline_report_snvs.yaml]: I actually think you should just put it back in

**Comment by apregier** [comment]
+1

**Comment by ernfrid** [comment]
Added back. If that looks good to you Allison then I will merge.

**Pull request merged**